README.md

```diff
@@ -1,5 +1,8 @@
 # charts-build-scripts
 
+This repository is maintained by `@rancher/release-team`.
+It is a crucial part of our release process for the Charts repository.
```

Comment:
```suggestion
It is a crucial part of our release process for the built-in [Helm Charts repository](https://github.com/rancher/charts).
```

Reply:
Since we have built-in Helm Charts and Partner Helm Charts, it is better to clarify it.